sdk/storage/azure-storage-blob/azure/storage/blob/models.py

```diff
@@ -1011,3 +1011,36 @@ def __str__(self):
 BlobPermissions.DELETE = BlobPermissions(delete=True)
 BlobPermissions.READ = BlobPermissions(read=True)
 BlobPermissions.WRITE = BlobPermissions(write=True)
+
+
+class UserDelegationKey(object):
```

Comment:
It seems this will be added by autorest, I will delete this in the second revision